javaslang-benchmark/pom.xml

```diff
@@ -36,17 +36,7 @@
         <dependency>
             <groupId>org.clojure</groupId>
             <artifactId>clojure</artifactId>
-            <version>1.9.0-alpha7</version>
```

Comment:
Please revert this commit as I stated before. Having these dependencies is of great relevance!